src/executor.rs

```diff
@@ -802,24 +802,10 @@ fn parse_result(
                 registry,
             ),
         },
-        CoreTypeConcrete::Felt252DictEntry(_) => todo!(),
```

Comment:
I know the variants are redundant with the catch-all branch below, but I feel that removing the explicit variants hides information that should be explicit, I would be more in favor of removing the catch-all if possible, or leaving as-is.